genestack_client/files_util.py

```diff
@@ -606,28 +606,27 @@ def find_files(
             offset=0,
             limit=MAX_FILE_SEARCH_LIMIT
     ):
+
         """
-        Search for files using filters.
+        Search for files using filters and return dictionary with next keys:
+
+         - total (int): total number of files on the platform matching the search filter
+         - result (list): list of file info dictionaries for the matching files between
+           ``offset`` and ``offset+limit``. See the documentation of
+           :py:meth:`~genestack_client.FilesUtil.get_infos` for the structure of these objects.
```

Comment:
This description is really unclear.  For starters, dict keys are ints and lists?